Source/WebCore/page/Quirks.cpp

```diff
@@ -1871,5 +1871,14 @@ bool Quirks::needsLaxSameSiteCookieQuirk(const URL& requestURL) const
     auto url = m_document->url();
     return url.protocolIs("https"_s) && url.host() == "login.microsoftonline.com"_s && requestURL.protocolIs("https"_s) && requestURL.host() == "www.bing.com"_s;
 }
+#if ENABLE(TEXT_AUTOSIZING)
+// rdar://127246368
+bool Quirks::skipTextAutoSizing() const
```

Comment:
Seems like the quirks seem to follow a naming convention of `needsX` or `shouldX` so I feel like we should do the same